client/Encounter/Encounter.ts

```diff
@@ -163,11 +166,15 @@ export class Encounter {
     return combatant;
   };
 
-  public AddCombatantFromStatBlock = (statBlockJson: {}, hideOnAdd = false) => {
+  public AddCombatantFromStatBlock = (
```

Comment:
Can you add a test for calling this with each of the new `variantMaximumHP` options?

Reply:
https://github.com/cynicaloptimist/improved-initiative/pull/442/files#diff-ff23d932f71d2b407cc8fcbe823d65cc